test/uk/gov/hmrc/tai/service/UpdateNextYearsIncomeServiceSpec.scala

```diff
@@ -161,6 +163,34 @@ class UpdateNextYearsIncomeServiceSpec extends PlaySpec with MockitoSugar {
     }
   }
 
+  "submit" must {
+    "post the values from cache to the tax account" in {
+      val nino = generateNino
+      val service = new UpdateNextYearsIncomeServiceTest
+
+      when(
+        service.journeyCacheService.currentCache(any())
+      ).thenReturn(
+        Future.successful(fullMap(employmentName, employmentId, employmentAmount))
+      )
+
+      when(
+        service.taxAccountService.updateEstimatedIncome(any(), any(), any(), any())(any())
```

Comment:
I think we should try to reduce the amount of anys we use and be more explicit on params.

Reply:
Agreed, in this case we're saying we don't care about the response from taxAccountService, only that it calls through to taxAccountService with the details